### ISSUE TITLE
Chef client cron recipe fails because of changes to cron_d resource

### ISSUE THREAD

**Issue by bigbam505** [chef-cookbooks/chef-client#352]
Version 1.7.1 was released for the cron cookbook [Difference Here](https://github.com/chef-cookbooks/cron/compare/v1.7.0...v1.7.1).  With this upgrade we started having the following issue:

```
Chef::Exceptions::ValidationFailed: cron_d[chef-client] (chef-client::cron line 105) had an error: Chef::Exceptions::ValidationFailed: Required argument command is missing!
```

It looks like the resource needs the command attributes from their code.  I don't see what has changed in this cookbook to start this other than them just re organizing their files and format.

**Comment by kbaltrinic** [comment]
Here is a more complete stack trace:

          ================================================================================
           Error executing action `delete` on resource 'cron_d[chef-client]'
           ================================================================================
           
           Chef::Exceptions::ValidationFailed
           ----------------------------------
           command is required
           
           Cookbook Trace:
           ---------------
           /tmp/kitchen/cache/cookbooks/cron/providers/d.rb:57:in `block in create_template'
           /tmp/kitchen/cache/cookbooks/cron/providers/d.rb:45:in `create_template'
           /tmp/kitchen/cache/cookbooks/cron/providers/d.rb:26:in `block in class_from_file'
           
           Resource Declaration:
           ---------------------
           # In /tmp/kitchen/cache/cookbooks/chef-client/recipes/cron.rb
           
           105:   cron_d 'chef-client' do
           106:     action :delete
           107:   end
           108:

**Comment by tas50** [comment]
This PR to cron should fix this bug.  I'll see if I can get it released this afternoon 

https://github.com/chef-cookbooks/cron/pull/66

**Comment by tas50** [comment]
This should be fixed in the cron 1.7.2 release
